docs/ABOUT.md

```diff
@@ -1,14 +1,5 @@
-R is a free, open source, programming language and software environment for statistical computing and graphics developed at Bell Laboratories as a GNU project. It is designed as a true computer language, and is written in C, Fortran and R.
+R is an open source programming language for data analysis and statistical computing. It was created by Ross Ihaka and Robert Gentleman at the University of Auckland, and is an implementation of the S programming language combined with lexical scoping semantics inspired by Scheme. The language is interpreted, dynamically typed, and has a strong foundation in functional programming. This makes it well suited to many of the challenges of data analysis.
```

Comment:
This paragraph feels a bit heavy, for some reason. I like the opening sentence, but then my brain starts having trouble making a coherent picture out of the words.

Do you happen to know what pain points the authors were trying to solve?

For example, with Ruby, the creator wanted to make something that makes programmers happy. He took all his favorite parts of his favorite languages and his goal was _delight_.

With Go, on the other hand, the problem was the immense complexity and long compile times of C++. The authors wanted something that was far simpler, but still had the power and performance of C.